cmd/telegraf/telegraf_windows.go

```diff
@@ -135,7 +136,11 @@ func (t *Telegraf) runAsWindowsService() error {
 			return fmt.Errorf("E! " + err.Error())
 		}
 	} else {
-		logger.SetupLogging(logger.LogConfig{LogTarget: logger.LogTargetEventlog})
+		err = logger.SetupLogging(logger.LogConfig{LogTarget: logger.LogTargetEventlog})
+		if err != nil {
+			return fmt.Errorf("E! " + err.Error())
```

Comment:
```suggestion
			return err.Error()
```

This `E!` should also be added by main

Reply:
@sspaink Should I do the same with every single `return fmt.Errorf("E! ... ")` inside `func (t *Telegraf) runAsWindowsService() error {`?

Reply:
@sspaink Done